homeworks/andrey.makarenko_Baksckua/5-arrays/arrays.js

```diff
@@ -0,0 +1,14 @@
+function height(tree) {
```

Comment:
1) The function name should have a verb. For example, `updateUser`, `fetchEntity`, `findLowestNumber`, `showTicket`, `displayName`.

`height` is a bad name for a function, because it doesn't describe what the function does and there is no verb in the name. Please, find a new one.

2) `tree` is also not a valid name for the tree height argument. It would be sound better if you name it `height`

Reply:
Done!